tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -61,31 +61,11 @@ def cert_data(self):
 
     @pytest.fixture(scope="module")
     def cert_setup(self, cert_data):
-        # Need a subdirectory under ssl-build with same name as Capsule name
-        with get_connection(timeout=100) as connection:
-            connection.run('mkdir ssl-build/{}'.format(cert_data['capsule_hostname']))
-            # Ignore creation error, but assert directory exists
-            assert connection.run('test -e ssl-build/{}'.format(cert_data['capsule_hostname']))
-        upload_file(
-            local_file=get_data_file('generate-ca.sh'),
-            remote_file="generate-ca.sh",
+        run_command(
+            'wget https://raw.githubusercontent.com/ekohl/ownca/master/ownca && chmod +x ownca'
```

Comment:
Hi, why do you want to change from the use of a locally stored script to downloading the script from some ones github page? I thought we are supposed to make things more reliable by reducing dependence on external servers especially personal pages.

Reply:
Good call @swadeley, I don't agree with this change. it points to a dynamic git ref, so the content could change without our knowing.

Reply:
will revert this, we are using the only the content from the repo itself. I will find the way to fix this in our script if possible. If not we can definitely use the fix tag version from forked repo ?

Reply:
By that way it can not be dynamic but stable manage by qe.

Reply:
updated the script !